### PR TITLE
Added in the necessary scripts to retrieve / reformat data sources.

### LINKED ISSUE #3972: Funcotator - need to consolidate data source massaging scripts
All scripts involved in massaging input data sources need to be put in the scripts directory.

### PR DESCRIPTION
**Pull request by jonn-smith**
Added unit tests for XsvLocatableTableCodecUnitTest and XsvTableFeature.

Fixes #3972

### CODE REVIEW

File: scripts/funcotator/data_sources/cosmic/GenericTsvReader.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+"""
```
Comment (LeeTL1220) [inline scripts/funcotator/data_sources/cosmic/GenericTsvReader.py]: Get rid of the `.pyc` file.

File: scripts/funcotator/data_sources/cosmic/createSqliteCosmicDb.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```
Comment (LeeTL1220) [inline scripts/funcotator/data_sources/cosmic/createSqliteCosmicDb.sh]: Can we have some usage docs?

File: scripts/funcotator/data_sources/cosmic/createTestDataForCosmic.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```
Comment (LeeTL1220) [inline scripts/funcotator/data_sources/cosmic/createTestDataForCosmic.sh]: Usage docs.

File: scripts/funcotator/data_sources/cosmic/shared_utils.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        buf_size = 1024 * 1024
+        read_f = f.read # loop optimization
+
+        buf = read_f(buf_size)
+        while buf:
+            lines += buf.count('\n')
+            buf = read_f(buf_size)
+
+    return lines
+
```
Comment (LeeTL1220) [inline scripts/funcotator/data_sources/cosmic/shared_utils.py]: Delete all `.pyc` files

Comment (LeeTL1220) [reply scripts/funcotator/data_sources/cosmic/shared_utils.py]: And put them on the ignore list.

File: scripts/funcotator/data_sources/getGencodeXRefseq.sh [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+hg19db="homo_sapiens_core_75_37"
+hg38db="homo_sapiens_core_90_38"
+
+hg19FileName=${outFileBaseName}_v75_37.hg19${outExt}
+hg38FileName=${outFileBaseName}_v90_38.hg38${outExt}
+
+################################################################################
+
+
+# Create our query to the DB:
```
Comment (LeeTL1220) [inline scripts/funcotator/data_sources/getGencodeXRefseq.sh]: If we don't need what is in the comment, please delete.  If we do, please explain what it is.

File: src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java [hunk 6]
```diff
@@ -72,8 +72,8 @@ public void exhaustiveArgumentTest(final String gtfFileName,
 
         final String outFileName = "funcotator_tmp_out_" + xsvMatchType.toString() + "_" + xsvMatchColumn + "_" + transcriptName + ".vcf";
 
-//        final File outputFile = createTempFile(outFileName.substring(0,outFileName.length()-4), outFileName.substring(outFileName.length()-4));
-        final File outputFile = new File(outFileName);
+        final File outputFile = createTempFile(outFileName.substring(0,outFileName.length()-4), outFileName.substring(outFileName.length()-4));
+//        final File outputFile = new File(outFileName);
```
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java]: Please remove commented line.

File: scripts/funcotator/data_sources/cosmic/GenericTsvReader.py (hunk 1 above)
Comment (jonn-smith) [reply scripts/funcotator/data_sources/cosmic/GenericTsvReader.py]: Fixed!

File: scripts/funcotator/data_sources/cosmic/createSqliteCosmicDb.sh (hunk 2 above)
Comment (jonn-smith) [reply scripts/funcotator/data_sources/cosmic/createSqliteCosmicDb.sh]: Sure.  FYI, these aren't really intended to be used frequently - they should only need to be run when updating data sources.

File: scripts/funcotator/data_sources/cosmic/createTestDataForCosmic.sh (hunk 3 above)
Comment (jonn-smith) [reply scripts/funcotator/data_sources/cosmic/createTestDataForCosmic.sh]: Fixed!

File: scripts/funcotator/data_sources/getGencodeXRefseq.sh (hunk 5 above)
Comment (jonn-smith) [reply scripts/funcotator/data_sources/getGencodeXRefseq.sh]: Fixed!

File: src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java (hunk 6 above)
Comment (jonn-smith) [reply src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java]: Fixed!

File: scripts/funcotator/data_sources/cosmic/shared_utils.py (hunk 4 above)
Comment (jonn-smith) [reply scripts/funcotator/data_sources/cosmic/shared_utils.py]: All `.pyc` files are now gone.

**Pull request merged**